### PR TITLE
done with 6th module assignment

### PR DESCRIPTION
**Pull request by Deepak1124**

### CODE REVIEW

File: module-6/filterDigitsAndAlphabets.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# [1, “a”, 2, “b”, 3, “c”] filter digits and alphabets separately using inbuilt
+# functions like map, filter or reduce
+
+
+data = [1, "a", 2, "b", 3, "c"]
+
+digits = []
+alphabets = []
+
+for ele in data:
```
Comment (tanmaythole) [inline module-6/filterDigitsAndAlphabets.py]: You need to use of inbuilt functions like map, filter, or reduce to implement this question

File: module-6/getUniqueNos.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# [1,2,3,1,3,4,6,5,3] get unique nums from list with basic constructs
+
```
Comment (tanmaythole) [inline module-6/getUniqueNos.py:2]: Try alternative solution for this, using set

File: module-6/replaceDigitsfromListOfStrings.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
+
+# [“name512”, “same1example”, “joy18full”] replace the digits from string inside list
+
```
Comment (tanmaythole) [inline module-6/replaceDigitsfromListOfStrings.py:3]: You can use regex here

**Review by tanmaythole — CHANGES_REQUESTED** [review]

File: module-6/getUniqueNos.py (hunk 2 above)
Comment (Deepak1124) [reply module-6/getUniqueNos.py:2]: done

File: module-6/filterDigitsAndAlphabets.py (hunk 1 above)
Comment (Deepak1124) [reply module-6/filterDigitsAndAlphabets.py]: done

File: module-6/replaceDigitsfromListOfStrings.py (hunk 3 above)
Comment (Deepak1124) [reply module-6/replaceDigitsfromListOfStrings.py:3]: done

**Review by tanmaythole — APPROVED** [review]

**Pull request merged**